src/main/java/seedu/address/logic/commands/FilterPropertyCommand.java

```diff
@@ -0,0 +1,82 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_LTE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TYPE;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.property.MatchingPrice;
+import seedu.address.model.property.Type;
+
+/**
+ * Filters properties based on their optional {@code PropertyType}.
+ */
+public class FilterPropertyCommand extends Command {
+    /** The command word to trigger the filtering action. */
+    public static final String COMMAND_WORD = "filterproperty";
+
+    /**
+     * Usage information for the filterclient command.
```

Comment:
small nit: could you update the javadoc to match with your command? here it's using the client filter docs